pylint/checkers/exceptions.py

```diff
@@ -288,6 +297,18 @@ class ExceptionsChecker(checkers.BaseChecker):
 
     def open(self) -> None:
         self._builtin_exceptions = _builtin_exceptions()
+        for exc_name in self.linter.config.overgeneral_exceptions:
+            if "." not in exc_name:
+                warnings.warn_explicit(
+                    "Specifying exception names in the overgeneral-exceptions option"
+                    " without module name is deprecated and support for it"
+                    " will be removed in pylint 3.0."
+                    f" Use fully qualified name (maybe 'builtins.{exc_name}' ?) instead.",
+                    category=UserWarning,
+                    filename="pylint: Command line or configuration file",
+                    lineno=1,
+                    module="pylint",
+                )
```

Comment:
With `--jobs 2`, the `open()` call is executed for each opened file. E.g. for HomeAssistant I got over 16.000 warnings just for 3 old config values. That should be fixed before we release `2.16`.
https://github.com/cdce8p/ha-core/actions/runs/3470026809/jobs/5797737991#step:8:25